### PR TITLE
Update GCP Dataproc custom image instruction and sample

### PR DESCRIPTION
**Pull request by mengdong**
Update instruction and example to create GCP dataproc custom image

### CODE REVIEW

**Comment by mengdong** [comment]
@aroraakshit  @viadea @sameerz FYI.

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -174,84 +174,18 @@ In the future, users will be able to provision a Dataproc cluster through Datapr
 can use example [pyspark notebooks](../demo/GCP/Mortgage-ETL-GPU.ipynb) to experiment.
 
 ## Build custom dataproc image to accelerate cluster init time
-In order to accelerate cluster init time to 4-5 minutes, we need to build a custom Dataproc image
-that already has NVIDIA drivers and CUDA toolkit installed. In this section, we will be using [these
+In order to accelerate cluster init time to 3-4 minutes, we need to build a custom Dataproc image
+that already has NVIDIA drivers and CUDA toolkit installed, with RAPIDS deployed. The custom image 
+could also be used in a air gap environment. In this section, we will be using [these
```
Comment (jlowe) [inline docs/get-started/getting-started-gcp.md:179]:
```suggestion
could also be used in an air gap environment. In this section, we will be using [these
```

File: docs/get-started/getting-started-gcp.md [hunk 2]
```diff
@@ -292,121 +226,16 @@ See [here](https://cloud.google.com/dataproc/docs/guides/dataproc-images#running
...
 [here](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions) for dataproc version description.
 
-The image `sample-207-ubuntu18-gpu-t4` is now ready and can be viewed in the GCP console under
+The image `sample-209-ubuntu18-gpu-t4` is now ready and can be viewed in the GCP console under
 `Compute Engine > Storage > Images`. The next step is to launch the cluster using this new image and
 new initialization actions (that do not install NVIDIA drivers since we are already past that step).
 
-Here is the new custom GPU initialization action that only configures YARN, the YARN node manager,
-GPU isolation and GPU exclusive mode:
+Here is the new custom GPU initialization action that needed to be ran after cluster creation with custom images, save this as `addon.sh`
```
Comment (jlowe) [inline docs/get-started/getting-started-gcp.md:233]:
```suggestion
Here is the new custom GPU initialization action that needs to be run after cluster creation with custom images, save this as `addon.sh`
```

File: docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh [hunk 3]
```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
```
Comment (jlowe) [inline docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh:1]: Missing NVIDIA copyrights and Apache license header

**Review by mengdong — COMMENTED** [review]
added suggestion to batch, added license header

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Review by aroraakshit — APPROVED** [review]

**Comment by aroraakshit** [comment]
Note: After building the custom image, the verification process fails. Although the cluster still launches successfully.

**Review by sameerz — APPROVED** [review]

**Pull request closed without merging**